R/management_defaults.R

```diff
@@ -107,6 +112,11 @@ add_management <- function(ID,B_LU_BRP, B_SOILTYPE_AGR,
     dt[is.na(M_GREEN), M_GREEN := TRUE]
     dt[is.na(M_NONBARE), M_NONBARE := FALSE]
     
+    # force green manure for potato and maize cultivations on sand
+    dt[soiltype.n == 'zand' & 
+         crop_name_scientific %in% c('solanum tuberosum', 'zea mays'),
```

Comment:
To be aligned with Dutch legislation use BRP codes instead of scientific names. Using scientific names introduces an additional layer that requires correct naming for BRP codes

Reply:
Alright, I thought this would have been more robust in case new potato or maize cultivation codes are added but I'll try and catch such a situation with a unit test.